microsoft-edge/web-platform/release-notes/135.md

```diff
@@ -0,0 +1,462 @@
+
+
+<!-- ---------- -->
+#### HTML features
+
+
+<!-- ---------- -->
+###### `command` and `commandfor` HTML attributes
+
+The `command` and `commandfor` HTML attributes for `<button>` elements allow you to assign behavior to buttons in an accessible and declarative way, while reducing how often a JavaScript page must be used to provide interactivity.
+
+When clicked, touched, or activated via a keypress, a button with the `commandfor` and `command` attributes dispatches a `CommandEvent` on the element that's referenced by `commandfor`, with some default behaviors, such as opening dialogs and popovers.
+
+
+<!-- ---------- -->
+###### Link `rel=facilitated-payment` to support push payments
+
+The `<link rel="facilitated-payment" href="...">` element acts as a hint that the browser uses to notify registered payment clients about a pending push payment.
+
+
+<!-- ---------- -->
+###### Support for `rel` and `relList` attributes in SVG `<a>` elements
+
+The SVGAElement interface in SVG 2.0 allows manipulating SVG `<a>` elements, which are similar to HTML anchor elements.  This change adds support for the `rel` and `relList` attributes, to enhance security and privacy for your webpages.
+
+This approach aligns with how HTML anchor elements work, and ensures consistency and ease of use across SVG and HTML.
+
+
+<!-- ---------- -->
+#### Web APIs
+
+
+<!-- ---------- -->
+###### Add MediaStreamTrack support to the Web Speech API
+
+The Web Speech API is a web standard API that allows you to incorporate speech recognition and synthesis into your webpages.
+
+The Web Speech API uses the user's default microphone as the default audio input.  MediaStreamTrack support allows a website to use the Web Speech API to caption other sources of audio, including remote audio tracks.
+
+
+<!-- ---------- -->
+###### Partitioning `:visited` links history
+
+To eliminate user browsing history leaks, HTML `<a>` elements are now styled as `:visited` only if they have been clicked from the top-level site and frame origin before.  This means that Microsoft Edge now partitions visited links according to three keys:
+* Link URL.
+* Top-level site.
+* Frame origin.
+
+By only styling links that have been clicked on this site and frame before, side-channel attacks that have been developed to obtain `:visited` links styling information are now obsolete.
+
+_Self-links_ are excluded from this; links to a site's own pages can be styled as `:visited` even if they have not been clicked on in this exact top-level site and frame origin before.  This exemption is only enabled in top-level frames or subframes which are same-origin with the top-level frame.
+
+
+<!-- ---------- -->
+###### Create service worker client and inherit service worker controller for iframe with `srcdoc`
+
+Documents created by iframes that use the `srcdoc` attribute are now service worker clients that are covered by their parent's service worker.<!-- todo: change plural to singular for clarity -->
```

Comment:
rewrite, changing plural to singular for clarity: 
Documents created by iframes that use the `srcdoc` attribute are now service worker clients that are covered by their parent's service worker.